lib/pub_sub/message.rb

```diff
@@ -13,18 +13,17 @@ def initialize(payload)
     def process
       begin
         validate_message!
-      rescue PubSub::ServiceUnknown => e
-        return
+        handler.process(data)
```

Comment:
Nvm, I see. You're catching all errors instead. What's the overall impact of this change. Will we lost the messages permanently if `handler.process` errors out?